### PR TITLE
fix: Update containerfile to check for presence of cosign.pub

### PR DESCRIPTION
**Pull request by bayou-brogrammer**

### CODE REVIEW

**Comment by bayou-brogrammer** [comment]
```Containerfile
...
ARG RECIPE=config/recipe.yml
ARG IMAGE_REGISTRY=ghcr.io/ublue-os
COPY cosign.pub /usr/share/ublue-os/cosign.pub
...
```

File: Cargo.toml [hunk 1]
```diff
@@ -20,27 +20,28 @@ colorized = "1"
 derive_builder = "0.13"
 directories = "5"
 env_logger = "0.11"
+format_serde_error = "0.3.0"
 futures-util = { version = "0.3", optional = true }
 fuzzy-matcher = "0.3"
 indexmap = { version = "2", features = ["serde"] }
+lazy_static = "1"
```
Comment (gmpinder) [inline Cargo.toml:27]: This is a wicked cool crate

File: cosign.pub [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE7lh7fJMV4dBT2jT1XafixUJa7OVA
+cT+QFVD8IfIJIS/KBAc8hx1aslzkH3tfeM0cwyCLB7kOStZ4sh6RyFQD9w==
+-----END PUBLIC KEY-----
```
Comment (gmpinder) [inline cosign.pub:4]: Was this supposed to be added in here?

Comment (bayou-brogrammer) [reply cosign.pub:4]: Oh poop. This was my test cosign file lol

File: Cargo.toml (hunk 1 above)
Comment (gmpinder) [reply Cargo.toml:27]: Wait where is this being used?

**Review by gmpinder — CHANGES_REQUESTED** [review]
We should try to keep dep changes to PRs that need them.

**Review by gmpinder — APPROVED** [review]

**Pull request merged**